src/genie/libs/parser/iosxe/tests/ShowCts/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,264 @@
+            },
+            "10.100.123.8": {
+                "source_ip": "192.168.2.24",
+                "conn_status": "On",
+                "duration": {
+                    "value": "20:12:40:45",
+                    "value_format": "dd:hr:mm:sec"
+                }
+            },
+            "10.100.123.9": {
+                "source_ip": "192.168.2.24",
+                "conn_status": "On",
+                "duration": {
+                    "value": "44:18:58:51",
+                    "value_format": "dd:hr:mm:sec"
+                }
+            },
+            "10.100.123.10": {
+                "source_ip": "192.168.2.24",
+                "conn_status": "On",
+                "duration": {
+                    "value": "44:18:58:51",
+                    "value_format": "dd:hr:mm:sec"
+                }
+            },
+            "10.100.123.11": {
+                "source_ip": "192.168.2.24",
+                "conn_status": "On",
+                "duration": {
+                    "value": "44:22:21:14",
+                    "value_format": "dd:hr:mm:sec"
+                }
+            },
+            "10.100.123.12": {
+                "source_ip": "192.168.2.24",
+                "conn_status": "On",
+                "duration": {
+                    "value": "44:18:58:51",
+                    "value_format": "dd:hr:mm:sec"
+                }
+            },
+            "10.100.123.13": {
+                "source_ip": "192.168.2.24",
+                "conn_status": "On",
+                "duration": {
+                    "value": "45:08:24:42",
+                    "value_format": "dd:hr:mm:sec"
+                }
+            },
+            "10.100.123.14": {
+                "source_ip": "192.168.2.24",
+                "conn_status": "On",
+                "duration": {
+                    "value": "45:08:24:42",
+                    "value_format": "dd:hr:mm:sec"
+                }
+            },
+            "10.100.123.15": {
+                "source_ip": "192.168.2.24",
+                "conn_status": "On",
+                "duration": {
+                    "value": "36:11:31:13",
+                    "value_format": "dd:hr:mm:sec"
+                }
+            },
+            "10.100.123.16": {
+                "source_ip": "192.168.2.24",
+                "conn_status": "On",
+                "duration": {
+                    "value": "36:12:13:54",
+                    "value_format": "dd:hr:mm:sec"
+                }
+            }
+        },
+        "total_connections": 16
+    },
+    "ip_sgt_bindings": {
+        "ipv4": {
+            "total_sxp_bindings": 3284,
+            "total_active_bindings": 3284
+        },
+        "ipv6": {
+            "total_sxp_bindings": 111,
+            "total_active_bindings": 111
+        }
+    },
+    "cts_role_based_enforcement": "Enabled",
+    "cts_role_based_vlan_enforcement": "Enabled",
+    "number_trusted_links": 0,
+    "number_untrusted_links": 0
```

Comment:
better to have level for respecting section (output with `====`)
```
'trusted_untrusted_links`: {
    "number_trusted_links": 0,
    "number_untrusted_links": 0

Reply:
Updated